### PR TITLE
[NVSHAS-9680] Adding NeuVector/Rancher SSO Permission Resources Documentation

### PR DESCRIPTION
**Pull request by sunilarjun**
Adding documentation of mapped permission resources associated with NeuVector/Rancher SSO. Tied to NVSHAS-9680.

### CODE REVIEW

File: docs/02.deploying/03.rancher/03.rancher.md [hunk 1]
```diff
@@ -74,6 +74,142 @@ Note in the above screen shot, two Rancher users admin and gkosaka have been aut
...
+Events
+Namespace
+RegistryScan
+RuntimePolicy
+RuntimeScan
+SecurityEvents
+SystemConfig
+```
+
+Resource display for Rancher `Global/Cluster` Role Template pages:
```
Comment (williamlin-suse) [inline docs/02.deploying/03.rancher/03.rancher.md:128]: Please remove lines 128~152
They are not available in Rancher's role creation -> resource drop-down menu

File: docs/02.deploying/03.rancher/03.rancher.md [hunk 2]
```diff
@@ -74,6 +74,142 @@ Note in the above screen shot, two Rancher users admin and gkosaka have been aut
...
+Events
+Namespace
+RegistryScan
+RuntimePolicy
+RuntimeScan
+SecurityEvents
+SystemConfig
+```
+
+Resource display for Rancher `Project` Role Template pages:
```
Comment (williamlin-suse) [inline docs/02.deploying/03.rancher/03.rancher.md:188]: Please remove lines 188~211
They are not available in Rancher's role creation -> resource drop-down menu

File: versioned_docs/version-5.4/02.deploying/03.rancher/03.rancher.md [hunk 3]
```diff
@@ -74,6 +74,142 @@ Note in the above screen shot, two Rancher users admin and gkosaka have been aut
...
+Events
+Namespace
+RegistryScan
+RuntimePolicy
+RuntimeScan
+SecurityEvents
+SystemConfig
+```
+
+Resource display for Rancher `Project` Role Template pages:
```
Comment (williamlin-suse) [inline versioned_docs/version-5.4/02.deploying/03.rancher/03.rancher.md:188]: Please remove lines 188~211
They are not available in Rancher's role creation -> resource drop-down menu

File: versioned_docs/version-5.4/02.deploying/03.rancher/03.rancher.md [hunk 4]
```diff
@@ -74,6 +74,142 @@ Note in the above screen shot, two Rancher users admin and gkosaka have been aut
...
+Events
+Namespace
+RegistryScan
+RuntimePolicy
+RuntimeScan
+SecurityEvents
+SystemConfig
+```
+
+Resource display for Rancher `Global/Cluster` Role Template pages:
```
Comment (williamlin-suse) [inline versioned_docs/version-5.4/02.deploying/03.rancher/03.rancher.md:128]: Please remove lines 128~152
They are not available in Rancher's role creation -> resource drop-down menu

**Comment by williamlin-suse** [comment]
Who will update https://open-docs.neuvector.com/ about the changes in the PR?

File: docs/02.deploying/03.rancher/03.rancher.md (hunk 1 above)
Comment (sunilarjun) [reply docs/02.deploying/03.rancher/03.rancher.md]: Thank you for reviewing, I've updated the content in this commit (https://github.com/neuvector/docs/pull/133/commits/423b3384a01d9b77f37dc3b502e47af701c8c3e7). The "Resources: NeuVector, Namespaced" is present for Rancher Global/Cluster Roles at lines 113-126.

**Comment by sunilarjun** [comment]
> Who will update https://open-docs.neuvector.com/ about the changes in the PR?

Thank you for reviewing! I've addressed the feedback in this commit: https://github.com/neuvector/docs/pull/133/commits/423b3384a01d9b77f37dc3b502e47af701c8c3e7

For clarification this PR is handling the content update for the documentation at https://open-docs.neuvector.com/, and I will be handling the port of this PR to the product docs (https://documentation.suse.com/cloudnative/security/5.4/en/overview.html) as well once approved.

**Review by williamlin-suse — APPROVED** [review]

**Pull request merged**